sdk/keyvault/azure-keyvault-certificates/README.md

```diff
@@ -29,102 +30,15 @@ authentication as demonstrated below.
 ### Prerequisites
 * An [Azure subscription][azure_sub]
 * Python 3.7 or later
-* A Key Vault. If you need to create one, you can use the
-[Azure Cloud Shell][azure_cloud_shell] to create one with these commands
-(replace `"my-resource-group"` and `"my-key-vault"` with your own, unique
-names):
-
-  (Optional) if you want a new resource group to hold the Key Vault:
-  ```sh
-  az group create --name my-resource-group --location westus2
-  ```
-
-  Create the Key Vault:
-  ```Bash
-  az keyvault create --resource-group my-resource-group --name my-key-vault
-  ```
-
-  Output:
-  ```json
-  {
-      "id": "...",
-      "location": "westus2",
-      "name": "my-key-vault",
-      "properties": {
-          "accessPolicies": [...],
-          "createMode": null,
-          "enablePurgeProtection": null,
-          "enableSoftDelete": null,
-          "enabledForDeployment": false,
-          "enabledForDiskEncryption": null,
-          "enabledForTemplateDeployment": null,
-          "networkAcls": null,
-          "provisioningState": "Succeeded",
-          "sku": { "name": "standard" },
-          "tenantId": "...",
-          "vaultUri": "https://my-key-vault.vault.azure.net/"
-      },
-      "resourceGroup": "my-resource-group",
-      "type": "Microsoft.KeyVault/vaults"
-  }
-  ```
-
-  > The `"vaultUri"` property is the `vault_url` used by [CertificateClient][certificate_client_docs]
+* An existing [Azure Key Vault][azure_keyvault]. If you need to create one, you can do so using the Azure CLI by following the steps in [this document][azure_keyvault_cli].
 
 ### Authenticate the client
-This document demonstrates using [DefaultAzureCredential][default_cred_ref]
-to authenticate as a service principal. However, [CertificateClient][certificate_client_docs]
-accepts any [azure-identity][azure_identity] credential. See the
-[azure-identity][azure_identity] documentation for more information about other
-credentials.
-
-#### Create a service principal (optional)
-This [Azure Cloud Shell][azure_cloud_shell] snippet shows how to create a
-new service principal. Before using it, replace "your-application-name" with
-a more appropriate name for your service principal.
-
-Create a service principal:
-```Bash
-az ad sp create-for-rbac --name http://my-application --skip-assignment
-```
+In order to interact with the Azure Key Vault service, you will need an instance of a [CertificateClient][certificate_client_docs], as well as a **vault url** (which you may see as "DNS Name" in the Azure Portal) and a credential object. This document demonstrates using a [DefaultAzureCredential][default_cred_ref], which is appropriate for most scenarios, including local development and production environments. We recommend using a [managed identity][managed_identity] for authentication in production environments.
```

Comment:
Does the vault URL appear as DNS Name for Key Vault as well? I only had that for Managed HSM based on some docs I copied from .NET a while ago.

Reply:
Good point; it doesn't appear to. Fixed!